packages/core/src/Slider/index.tsx

```diff
@@ -340,7 +340,17 @@ export const Slider: React.FC<SliderProps> = ({
     }
   }, [tickConfig])
 
-  const fraction = (value - min) / (max - min)
```

Comment:
Perhaps just `Math.max(0, Math.min(1, (value - min) / (max - min)))`, or use the `clamp` function in this file already. `clamp((value - min) / (max - min))`?

Reply:
Done